plugins/modules/site_workflow_manager.py

```diff
@@ -2042,17 +2091,13 @@ def get_diff_deleted(self, config):
         if self.compare_dnac_versions(self.get_ccc_version(), "2.3.5.3") <= 0:
             site_exists = self.have.get("site_exists")
             site_name_hierarchy = self.want.get("site_name_hierarchy")
-
+            site_id = self.have.get("site_id")
             if not site_exists:
-                self.status = "success"
-                self.site_absent_list.append(site_name_hierarchy)
-                self.log(
-                    "Unable to delete site '{0}' as it's not found in Cisco Catalyst Center".format(self.want.get("site_name_hierarchy")), "INFO")
+                if site_name_hierarchy not in self.deleted_site_list:
+                    self.site_absent_list.append(site_name_hierarchy)
+                self.log("Unable to delete site '{0}' as it's not found in Cisco Catalyst Center".format(site_name_hierarchy), "INFO")
```

Comment:
self.log(
        "Failed to delete site '{0}'. Reason: The site was not found in the Cisco Catalyst Center.".format(site_name_hierarchy),
        "DEBUG"
    )